controllers/temprole_controller.py

```diff
@@ -123,6 +124,32 @@ async def view_temproles(user: User, interaction: Interaction):
 
         await interaction.response.send_message(response, ephemeral=True)
 
+    @staticmethod
+    async def view_users(role: Role, interaction: Interaction):
+        temprole_users = DB().get_temprole_users(role.id, interaction.guild_id)
+
+        if len(temprole_users) == 0:
+            return await interaction.response.send_message(
+                f"`@{role.name}` is not currently assigned to any users as a temprole!",
+                ephemeral=True,
+            )
+        
+        title = f"Users with `@{role.name}` temprole:"
+        user_list = []
+        for user in temprole_users:
+            member = interaction.guild.get_member(user.user_id)
+            if member is None:
+                response = f"Could not find user {user.user_id}"
```

Comment:
Add newline here to preserve formatting when some of the results cannot be found:
<img width="641" alt="image" src="https://github.com/crscillitoe/RoboBanana/assets/17772186/d6aebc79-80d7-428e-b823-852e23b44ff9">